src/main/cli.py

```diff
@@ -6,280 +6,224 @@
+                        help="the path of commit message file")
+    args = parser.parse_args()
 
-
-def run_hook(msg: str):
-    """
-    Run the main logic of the hook.
-
-    If one of the checks failed abort commit(exit nonzero),
-    display appropriate errors and hints.
-    Otherwise exit zero and allow the `commit` command
-
-    Args:
-        msg (str): The commit message to check.
-    """
-    global IS_EXIT_1
-    check_length(msg)
-    check_subject_line(msg)
-    check_body(msg)
-    if IS_EXIT_1:
-        show_example()
+    global default_prefixes
+    default_prefixes = default_prefixes + args.prefix
+    msg = read_msg(args.path)
+    if not msg.strip():
+        print(f"ֿ{RED}ERROR: commit message can't be empty!{OFF}\n")
         sys.exit(1)
-    else:
-        print(f"\n{GREEN}\
-CONGRATS! Commit message matches the chaos-hub commit rules 👍\n{DEFAULT}")
-        sys.exit(0)
-
-
-def show_example():
-    """
-    Display a commit message example.
-
-    The example follows chaos-hub team commit rules.
-    """
-    print(f"{GREEN}\n\
-EXAMPLE:\n\
-  Refactor foo function in ...\n{BLUE}\
-<body is optional, adding it leave an empty line here>\n{GREEN}\
-  - Fix ...\n\
-  - Add ...\n\
-  - Remove ... \n{YELLOW}\
-HINT: to read chaos-hum team rules visit: {BLUE}\
-https://github.com/dimaka-wix/commit-msg-hook.git \n{DEFAULT}")
+    run_hook(msg)
 
 
-def get_commit_msg() -> str:
+def read_msg(path: str) -> str:
     """
     Get commit message content.
 
-    Try to get the message on the path given in a command line.
-    If fail, abort commit(exit nonzero), display appropriate error and hint.
-    If there are no arguments, show information about the hook.
+    Try to get the message on the given path. If fail, abort commit(exit nonzero), display appropriate error and hint.
+
+    Args:
+        path (str): The path of the file with commit message.
 
     Returns:
         str: The commit message.
     """
-    args = sys.argv
-    if len(args) < 2:
-        print(f"\n{GREEN}\
-This hook is made as custom plugins under the {BLUE}https://pre-commit.com \
-{GREEN}hook framework\nIt checks if commit message \
-matches the chaos-hub team commit rules\n{YELLOW}\
-HINT: to read chaos-hum team rules visit {BLUE}\
-https://github.com/dimaka-wix/commit-msg-hook.git \n{DEFAULT}")
-        sys.exit(0)
-    path = args[len(args) - 1]
-    path = COMMIT_EDITMSG if path.upper() in COMMIT_EDITMSG else path
     try:
-        with open(path, "r", encoding="utf-8") as msg:
-            commit_msg = msg.read()
+        with open(path, "r", encoding="utf-8") as file:
+            msg = file.read()
     except FileNotFoundError:
         print(f"\n{RED}\
-ERROR: file '{path}' not found!\n{YELLOW}\
-HINT:  the commit message is usually saved in {COMMIT_EDITMSG}\n{DEFAULT}")
+ERROR: the path '{path}' not found!\n{YELLOW}\
+HINT:  the commit message is usually saved in {COMMIT_EDITMSG}{OFF}\n")
```

Comment:
In this scenario, you can use string.format() instead of using the f string because in this way it's easier to see all the variables and also it can be less written in one line, so you won't need to do all the formating yourself with all the '/' and the new lines.